pandas/tests/window/test_numba.py

```diff
@@ -121,3 +121,71 @@ def test_invalid_kwargs_nopython():
         Series(range(1)).rolling(1).apply(
             lambda x: x, kwargs={"a": 1}, engine="numba", raw=True
         )
+
+
+@td.skip_if_no("numba", "0.46.0")
+@pytest.mark.filterwarnings("ignore:\\nThe keyword argument")
+# Filter warnings when parallel=True and the function can't be parallelized by Numba
+class TestTableMethod:
+    def test_table_series_valueerror(self):
+        def f(x):
+            return np.sum(x, axis=0) + 1
+
+        with pytest.raises(
+            ValueError, match="method='table' not applicable for Series objects."
+        ):
+            Series(range(1)).rolling(1, method="table").apply(
+                f, engine="numba", raw=True
+            )
+
+    def test_table_method_rolling(
```

Comment:
you mean this test has a large number of possibilities? can you use --duration and run locally, maybe need to cut down on the side / parameterization here.

Reply:
Yeah I removed the `center` and `closed` parameterization and that helped. (Dont really need to test these keywords anyways because they are indexer related and out of this PR scope)